CITATION.cff

```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+title: NOMAD Simulations Schema Plugin
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Nathan
+    family-names: Daelman
+    orcid: 'https://orcid.org/0000-0002-7647-1816'
+  - given-names: Bernadette
+    family-names: Mohr
+    orcid: 'https://orcid.org/0000-0003-0903-0073'
+  - given-names: José M.
```

Comment:
I'd like to be main author and you (@JFRudzinski) at the end

Reply:
yeah, I was going to do that at first. Ok, then I leave the others in alphabetical order in the middle?

Reply:
I think it makes sense to add them in order of contribution.

would you mind adding Alvin as well?

Reply:
@ladinesa FYI - I am adding you as author in our Zenodo publication